### PR TITLE
[clients] - improve error entries messages

### PR DESCRIPTION
**Pull request by GuyAfik**
## Related Issues
fixes:

## Description
* align the error messages to fit both for a command executed in a playground and a command executed in a playbook's task, use the same code for both cases.

### CODE REVIEW

File: demisto_sdk/commands/common/clients/xsoar/xsoar_api_client.py [hunk 1]
```diff
@@ -980,6 +982,37 @@ def run_cli_command(
...
+
+        Returns:
+            Formatted error entries
+        """
+        error_entries: Set[str] = set()
+
+        for entry in entries:
+            if entry.type == self._ENTRY_TYPE_ERROR and entry.parent_content:
+                # Checks for passwords and replaces them with "******"
+                parent_content = re.sub(
```
Comment (JudahSchwartz) [inline demisto_sdk/commands/common/clients/xsoar/xsoar_api_client.py:1000]: what are you checking here? Why is the entry printing passwords? Is this the only case we need to check for?

Comment (GuyAfik) [reply demisto_sdk/commands/common/clients/xsoar/xsoar_api_client.py:1000]: we talked about it previously [here](https://github.com/demisto/demisto-sdk/pull/3732#discussion_r1382542636)

Comment (JudahSchwartz) [reply demisto_sdk/commands/common/clients/xsoar/xsoar_api_client.py:1000]: are there other forms of authentication or other secrets that might be printed?

Comment (GuyAfik) [reply demisto_sdk/commands/common/clients/xsoar/xsoar_api_client.py:1000]: nope

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**